apps/azureserviceoperator-system/aso/aso.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: azure-service-operator
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: azure-service-operator
+      version: v2.0.0-beta.5
```

we should use the latest version:
```suggestion
      version: v2.2.0
```